app/models/schedule.rb

```diff
@@ -1,7 +1,6 @@
 class Schedule < ActiveRecord::Base
   include RankedModel
 
-  has_many :volunteers, through: :schedule_volunteers
```

Comment:
For context, it looks like `Schedule#schedule_volunteers` was removed with commit [13a22f6](https://github.com/boulder-food-rescue/food-rescue-robot/commit/13a22f6#diff-220f24c8b08c15da92dc4d1d17f8da76L5)